great_expectations/agent/models.py

```diff
@@ -14,13 +14,21 @@ class EventBase(AgentBaseModel):
 
 
 class RunOnboardingDataAssistantEvent(EventBase):
+    # TODO(https://greatexpectations.atlassian.net/browse/DX-652): Deprecate once event is no longer being emitted
     type: Literal[
         "onboarding_data_assistant_request.received"
     ] = "onboarding_data_assistant_request.received"
     datasource_name: str
     data_asset_name: str
 
 
+class RunDataAssistantEvent(EventBase):
+    type: Literal["data_assistant_request.received"] = "data_assistant_request.received"
```

Comment:
while we're updating this, could we remove the `.received`? We're not using these events as broadly as we imagined we might when we first spec'd this out, so that extra namespace is unnecessary.

Reply:
Thanks will update